python/README.md

```diff
@@ -85,6 +93,23 @@ def handler(event, context)
   ...
 ```
 
+**Fetching a pre-configured tracer anywhere**
+
+```python
+# handler.py
+from aws_lambda_powertools.tracing import Tracer
+tracer = Tracer(service="payment")
```

Comment:
Does this uses `POWERTOOLS_SERVICE_NAME` if I don't set service or is this mandatory?

Reply:
The former - Explicit first, env second.